mkdocs.yaml

```
@@ -169,10 +171,12 @@ nav:
     - 'models/home.md'
     - Fine-tuning: 'models/fine_tuning.md'
     - Benchmarks and Evaluations: 'models/benchmarks_and_evaluations.md'
-  - Assistants:
-    - Overview: 'assistants/overview.md'
-    - Integrate: 'assistants/integrate.md'
-  - API: 'api.md'
+  - Integration:
+    - Overview: 'integration/overview.md'
+    - Quickstart: 'integration/quickstart.md'
+  - APIs:
+    - Web API: 'https://api.near.ai/openapi.json'
+    - Library API: 'api.md'
```

```suggestion
    - Library API: 'api.md'
    - Web API: 'https://api.near.ai/openapi.json'
```

these need to be flipped otherwise when you click on API in the header it will only show the `openapi.json` file and you will not be able to see the Library API docs